### DISCUSSION TITLE
How can I find what plugin is slowing me down?

### DISCUSSION THREAD

**Discussion by oriongonza** [neovim/neovim#26049] [General]
This is the output of Lazy profiler. I'm opening a 7.7MB xml file.
![image](https://github.com/neovim/neovim/assets/47633543/d669dca5-798a-45ae-bfc9-7cf92911cf49)

What's happening at UIEnter that's taking so long?
if I open it with --clean it's instant
Also gg=G takes forever with plugins on large xml files but it's somewhat fast with --clean and I've no idea why. I really don't want to be binary searching disabling plugins
The file is not even that large, only 7.7MB

https://github.com/dev-ardi/init.lua This is my config. All plugins are in lua/plugins.lua

Also, having two instances crashes on windows (not on linux) Is my only option to bisect the plugins?

**Comment by clason** [comment]
use `nvim --startuptime profile.log`.

And, yes, your best and only option is to bisect your plugins. Nobody is going to do that for you.
